librocksdb_sys/crocksdb/c.cc

```diff
@@ -1944,6 +1948,27 @@ void crocksdb_options_destroy(crocksdb_options_t* options) {
   delete options;
 }
 
+void crocksdb_column_family_descriptor_destroy(
+    crocksdb_column_family_descriptor* cf_dec) {
```

Comment:
s/cf_dec/cf_desc

Reply:
Please replace the others too.

Reply:
list of crocksdb_column_family_descriptor calling cf_decs
single crocksdb_column_family_descriptor calling cf_dec

Reply:
s/dec/desc
s/decs/descs

Reply:
ok